src/pages/components/text-input/style.mdx

```diff
@@ -106,3 +98,11 @@ should be three words or less.
 </div>
 
 <Caption>Structure and spacing measurements for text area | px / rem</Caption>
+
+## Sizes 
+
+| Size        | Height (px / rem)  |
```

Comment:
- Add a column before the size column called "Element". The first row under the Element column can say "Input".